packages/docs-website/docs/protocol-tutorial/finalize-a-channel-happy.md

```diff
@@ -45,14 +45,14 @@ for (let i = 0; i < 3; i++) {
   participants[i] = wallets[i].address;
 }
 const chainId = '0x1234';
-const channelNonce = bigNumberify(0).toHexString();
+const channelNonce = BigNumber.from(0).toHexString();
```

Comment:
We should probably _not_ be making changes to any .md files in `docs-website/docs/protocol-tutorial` since the accompanying repo is still using ethers 4 

https://github.com/statechannels/nitro-tutorial/blob/2cf51f00ddf9b47d83685b3b14d9b58d24e6585d/package.json#L19

Reply:
Resolved in 4ba1f42